django/contrib/admindocs/templates/admin_doc/model_detail.html

```diff
@@ -48,6 +49,30 @@ <h2 class="subhead">{{ summary }}</h2>
 </table>
 </div>
 
+{% if methods %}
+<h3>Methods with arguments</h3>
+<div class="module">
+<table class="model">
+<thead>
+<tr>
+    <th>{% trans 'Method' %}</th>
+    <th>{% trans 'Arguments' %}</th>
+    <th>{% trans 'Description' %}</th>
+</tr>
+</thead>
+<tbody>
+{% for method in methods|dictsort:"name" %}
+<tr>
+    <td>{{ method.name }}</td>
+    <td>{{ method.arguments|join:', ' }}</td>
+    <td>{{ method.verbose }}{% if method.help_text %} - {{ method.help_text|safe }}{% endif %}</td>
```

Comment:
I don't think annotating a method with a "help_text" attribute is common -- that's mainly for model fields I think.